README.rst

```diff
@@ -0,0 +1,260 @@
+==============================
+Amazon SageMaker RL Containers
+==============================
+
+A set of Dockerfiles that provides Reinforcement Learning (RL) solutions to be used in SageMaker. 
+
+The SageMaker team uses this repository to build its official RL images. On how to use any of these images on SageMaker,
+see `Python SDK <https://github.com/aws/sagemaker-python-sdk>`__.
+For end users, this repository is typically of interest if you need implementation details of
+the official image, or if you want to use it to build your own customized RL image.
+
+For information on running RL jobs on SageMaker: `SageMaker RLEstimators
+<https://github.com/aws/sagemaker-python-sdk/tree/master/src/sagemaker/rl>`__.
+
+For notebook examples: `SageMaker Notebook
+Examples <https://github.com/awslabs/amazon-sagemaker-examples/tree/master/reinforcement_learning>`__.
+
+
+Table of Contents
+-----------------
+
+#. `Getting Started <#getting-started>`__
+#. `Building Your Image <#building-your-image>`__
+#. `Running the Tests <#running-the-tests>`__
+#. `RL Images Provided by SageMaker <#rl-images-provided-by-sagemaker>`__
+
+Getting Started
+---------------
+
+Prerequisites
+~~~~~~~~~~~~~
+
+Make sure you have installed all of the following prerequisites on your
+development machine:
+
+- `Docker <https://www.docker.com/>`__
+
+For Testing on GPU
+^^^^^^^^^^^^^^^^^^
+
+-  `Nvidia-Docker <https://github.com/NVIDIA/nvidia-docker>`__
+
+Recommended
+^^^^^^^^^^^
+
+-  A Python environment management tool (e.g.
+   `PyEnv <https://github.com/pyenv/pyenv>`__,
+   `VirtualEnv <https://virtualenv.pypa.io/en/stable/>`__)
+
+Building Your Image
+-------------------
+
+`Amazon SageMaker <https://aws.amazon.com/documentation/sagemaker/>`__
+utilizes Docker containers to run all training jobs and inference endpoints.
+
+The Docker images are built from the Dockerfiles specified in
+`coach/docker <https://github.com/aws/sagemaker-rl-container/tree/master/coach/docker>`__
+and `ray/docker <https://github.com/aws/sagemaker-rl-container/tree/master/ray/docker>`__.
+
+The Docker files are grouped based on RL toolkit (Coach or Ray), toolkit version and separated
+based on framework, e.g.: ``coach/docker/0.11.0/Dockerfile.mxnet``.
+
+
+All Dockerfiles use deep learning framework images provided by SageMaker as their "base" images.
+
+These "base" images are specified with the following naming convention:
+
+::
+
+    # 520713654638.dkr.ecr.<region>.amazonaws.com/sagemaker-<framework>:<framework_version>-<processor>-py3
+    520713654638.dkr.ecr.<region>.amazonaws.com/sagemaker-<tensorflow-scriptmode_or_mxnet>:<framework_version>-<cpu_or_gpu>-py3
```

Comment:
I'd "uncomment" the first line, and then turn the second line into an explanation that `framework` can be either `tensorflow-scriptmode` or `mxnet` and then the same for `processor`. The lone sentence below will then also flow a little better